src/status_im2/contexts/wallet/home/style.cljs

```diff
@@ -5,6 +5,9 @@
    :padding-top        8
    :padding-bottom     12})
 
+(def accounts-container
+  {:height 112})
```

Comment:
should we just use flex instead?

Reply:
Using flex will affect the height of the component and it will change every time we change assets/collectibles tab (because we are switching between different flatlists and they take different amounts of space depending on children).
That change will result in the whole tabbar jumping when navigating tabs.
Btw, I don't remember discussing any problem with having fixed height, especially on the screens like this, where flat-list at the bottom will eat up all the remaining space. I believe fixed with is more problematic. But maybe I miss something?

Reply:
Yeah you're right!